src/java.base/share/classes/sun/security/ssl/CipherSuite.java

```diff
@@ -972,19 +968,10 @@ static String nameOf(int id) {
             }
 
             boolean found = false;
-            for (CipherSuite cs : CipherSuite.values()) {
-                if (!cs.supportedProtocols.isEmpty()) {
-                    if (cs.name.equals(name) ||
-                            cs.aliases.contains(name)) {
-                        cipherSuites.add(cs);
-                        found = true;
-                        break;
-                    }
-                } else {
-                    // values() is ordered, remaining cipher suites are
-                    // not supported.
-                    break;
-                }
+            CipherSuite cs;
+            if ((cs = cipherSuiteNames.get(name)) != null && !cs.supportedProtocols.isEmpty()) {
```

Comment:
Nice update!  I appreciate if you could avoid lines longer than 80 characters, since they’re not handled well by many terminals and tools (See [Java Code Conventions](https://www.oracle.com/java/technologies/javase/codeconventions-indentation.html)).

Reply:
Thank you @XueleiFan, Updated the length of the line.

Reply:
Thank you for the update.  I have no further comment.